restructuredtext_lint/sphinx.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+
+import contextlib
+
+try:
+    import sphinx  # noqa
+
+    from sphinx.domains import c as c_domain
+    from sphinx.domains import cpp as cpp_domain
+    from sphinx.domains import javascript as javascript_domain
+    from sphinx.domains import python as python_domain
+    from sphinx.domains import std as std_domain
```

Comment:
This is a very limited list of languages. What lead to this choice and do we need them?